ome/conda-bftools
# BFTools 6.6.0

Replaces https://github.com/ome/conda-bftools/pull/5

To test:
* create a conda environment and install bftools
* Check command from bftools like ``showinf``

To install
```
conda create --name bftools
conda activate bftools
conda install -c jburel bftools
```
